packages/aragon-wrapper/src/index.js

```diff
@@ -214,20 +214,49 @@ export default class Aragon {
     const SET_PERMISSION_EVENT = 'SetPermission'
     const CHANGE_PERMISSION_MANAGER_EVENT = 'ChangePermissionManager'
 
-    const aclObservables = [
-      SET_PERMISSION_EVENT,
-      CHANGE_PERMISSION_MANAGER_EVENT
-    ].map(event =>
-      this.aclProxy.events(event)
-    )
-
-    // Set up permissions observable
+    const ACL_CACHE_KEY = `acl-${aclAddress}`
+
+    // Check if we have cached ACL for this address
+    // Cache object for an ACL:
+    // [ { permissions, blockNumber } ]
+    const cached = await this.cache.get(ACL_CACHE_KEY, {})
+    const { permissions: cachedPermissions, blockNumber: cachedBlockNumber } = cached
+
+    let events
```

Comment:
```suggestion
    const events  = this.aclProxy.events(null, cachedPermissions && cachedBlockNumber)
```

If we don't need the log in L227 we can just do this